docs: expand community docs: contributor - recommended tools (#1777)
asyncapi/community: docs/onboarding-guide/tools-and-setup.md

---
title: Recommended tools and setup
weight: 60
---

> This article covers the recommended tools you should have as a contributor and how to set up AsyncAPI website in your local environment.

Reviewer: ```suggestion
These are the recommended tools for contributors, as well as instructions for setting up the AsyncAPI website in your local environment.
```

---
title: Recommended tools and setup
weight: 60
---

This article covers the recommended tools you should have as a contributor and how to set up AsyncAPI website in your local environment.

## Recommended Tools

> These are the recommended tools:

Reviewer 2: This phrase is redundant
